OpenNMT/onmt/Models.py

```diff
@@ -99,9 +98,9 @@ def __init__(self, opt, dicts):
 
 
     def forward(self, input, hidden, context, init_output):
-        emb = self.word_lut(input)
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
The speed difference with the transposes seems pretty negligible on the demo data, but I still think it would be nice to avoid them where possible, so I'll look at switching it over to (batch, seq_len, dim) everywhere.